eos/rare-b-decays/Makefile.am

```diff
@@ -7,6 +7,7 @@ lib_LTLIBRARIES = libeosrarebdecays.la
 libeosrarebdecays_la_SOURCES = \
 	b-to-k-charmonium.cc b-to-k-charmonium.hh \
 	b-to-kstar-charmonium.cc b-to-kstar-charmonium.hh \
+        bs-to-phi-charmonium.cc bs-to-phi-charmonium.hh \
```

Comment:
This is going to cause trouble. There are spaces instead of tabs.

Reply:
My mistake, I don't know what caused this, sublime business I think. Anyway, corrected